rewrite.go

```diff
@@ -109,7 +109,14 @@ func rewriteGoFile(name, qual string, paths []string) error {
 // sep is the signature set of path elements that
 // precede the original path of an imported package
 // in a rewritten import path.
-const sep = "/Godeps/_workspace/src/"
+var sep = "/Godeps/_workspace/src/"
+var VendorExperiment = os.Getenv("GO15VENDOREXPERIMENT") == "1"
+
+func init() {
```

Comment:
I'm suggesting the first, e.g. `var sep = defaultSep(VendorExperiment)`.

Flags are defined in init functions only because they have to to avoid a cycle. (Try changing it to `var saveR = cmdSave.Flag.Bool(...)` to see what happens.)